packages/hydrogen/edge-entry.js

```diff
@@ -5,8 +5,10 @@ import indexTemplate from '__RELATIVE__/dist/client/index.html?raw';
 import { ReadableStream } from 'web-streams-polyfill/ponyfill';
 Object.assign(globalThis, { ReadableStream });
```

Comment:
We may want to do `globalThis` changes in one clear place up top, like:

```
Object.assign(globalThis, {
  // ReadableStream is bugged in Vercel Edge, overwrite with polyfill
  ReadableStream,

  // Hydrogen exposes env vars through `Oxygen.env`
  Oxygen: {
    env: process.env
  }
});
```

Reply:
I second this idea. Global is global and it's best outside of a function.

Reply:
👋 hey I made this change here https://github.com/vercel/vercel/pull/8496 if you want to take a look